readme.md

```diff
@@ -191,10 +191,13 @@ You can use the programmatic API to retrieve the diagnostics and do something wi
 import tsd from 'tsd';
 
 (async () => {
-	const diagnostics = await tsd();
+	const diagnoser = await tsd();
 
-	console.log(diagnostics.length);
-	//=> 2
+	// Returns the number of tests evaludated.
+	console.log(diagnoser.numTests)
```

Comment:
```suggestion
	console.log(diagnoser.testCount)
```

Reply:
Would you like the `numTests` to be renamed into `testCount`?

Reply:
Yes

Reply:
I've refactored all instances of `numTests` into `testCount`.